src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs

```diff
@@ -243,3 +246,24 @@ impl Sandbox for VirtSandbox {
         Ok(())
     }
 }
+
+#[async_trait]
+impl Persist for VirtSandbox {
+    type State = persist::sandbox::SandboxState;
+    type ConstructorArgs = ();
+
+    /// Returns the current state of the component.
+    async fn save(&self) -> Result<Self::State> {
+        let resource_state = self.resource_manager.save().await;
+        Ok(persist::sandbox::SandboxState {
+            resource: resource_state,
+        })
+    }
+
+    async fn restore(
+        _constructor_args: Self::ConstructorArgs,
+        _state: &Self::State,
+    ) -> Result<Self> {
+        todo!()
```

Comment:
Can you add an issue URL to the `todo!()` call?

Reply:
fixed